source/Calamari/Kubernetes/Conventions/Helm/HelmManifestAndStatusReporter.cs

```diff
@@ -42,12 +44,19 @@ public async Task StartBackgroundMonitoringAndReporting(
             await Task.Run(async () =>
                            {
                                var resourceStatusCheckIsEnabled = deployment.Variables.GetFlag(SpecialVariables.ResourceStatusCheck);
-                               
-                               if (resourceStatusCheckIsEnabled
+                      
+                               if (
+                                   resourceStatusCheckIsEnabled
                                    || FeatureToggle.KubernetesLiveObjectStatusFeatureToggle.IsEnabled(deployment.Variables)
                                    || OctopusFeatureToggles.KubernetesObjectManifestInspectionFeatureToggle.IsEnabled(deployment.Variables))
                                {
-                                   var manifest = await PollForManifest(deployment, helmCli, releaseName, revisionNumber);
+                                   if (!DoesHelmCliSupportManifestRetrieval(out var helmVersion))
+                                   {
+                                       log.Warn($"Octopus needs Helm v3.13 or later to display object status and manifests. Your current version is {helmVersion}. Please update your Helm executable or container to enable our new Kubernetes capabilities. Learn more in our {log.FormatShortLink("KOS", "documentation")}.");
+                                       return;
+                                   }
```

Comment:
@kevjt I realised this _had_ to go inside this `if` statement, otherwise we would be doing the version check even when manifest retrieval was disabled

Reply:
oh yea, good catch